tests/unit/labels/MulticlassLabels_unittest.cc

```diff
@@ -101,3 +102,16 @@ TEST_F(MulticlassLabels, multiclass_labels_from_dense_not_contiguous)
 	ASSERT_NE(converted2.get(), nullptr);
 	EXPECT_TRUE(converted2->get_labels().equals({0, 1, 1}));
 }
+
+TEST_F(MulticlassLabels, view)
+{
+	auto labels = some<CMulticlassLabels>(labels_true);
+	SGVector<index_t> subset{0, 2};
+	auto labels_subset = wrap(labels->view(subset)->as<CMulticlassLabels>());
+
+	ASSERT_EQ(labels_subset->get_num_labels(), subset.vlen);
+	for (auto i : range(subset.vlen))
+	{
+		EXPECT_EQ(labels_subset->get_int_label(i), labels_true[subset[i]]);
```

Comment:
maybe we can actually systematically test all access methods of labels and features somehow?
I would also assert that the view's data pointer is the same as the original one (i.e. no copy happened)

Reply:
we don't have a way to test different features/labels types, and different access methods now.
and we can't check view's data pointer because that's a private member, calling `get_labels` will create a copy if subset is present

Reply:
we don't have a way to test different features/labels types, and different access methods now.
and we can't check view's data pointer because that's a private member, calling `get_labels` will create a copy if subset is present